widgets/ContainerFrame.lua

```diff
@@ -769,6 +793,10 @@ local function FilterByBag(slotData)
 		name = REAGENT_BANK
 	elseif bag <= NUM_BAG_SLOTS then
 		name = format(L["Bag #%d"], bag)
+	elseif addon.isRetail then
+		if bag == REAGENTBAG then
+			name = format(L["Reagent Bag"])
+		end
```

Comment:
here name could be nil and it produces an error with the strjoin below. add name = format(L["Bank bag #%d"], bag - NUM_BAG_SLOTS)